### PR TITLE
ZH-trans: Update content/en/docs/tasks/inject-data-application/...

### PR DESCRIPTION
**Pull request by xichengliudui**
ZH-trans: Update content/en/docs/tasks/inject-data-application/...

### CODE REVIEW

**Comment by xichengliudui** [comment]
这些修改几乎都是 website 展示的问题，很少一些涉及翻译。

**Comment by xichengliudui** [comment]
file address:

https://v1-12.docs.kubernetes.io/zh/docs/tasks/inject-data-application/define-command-argument-container/
https://v1-12.docs.kubernetes.io/zh/docs/tasks/inject-data-application/distribute-credentials-secure/
https://v1-12.docs.kubernetes.io/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/

**Comment by xichengliudui** [comment]
preview 有些问题我解决一下。
/hold

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -34,34 +34,48 @@ content_template: templates/task
...
+       ```shell
        kubectl create -f https://k8s.io/docs/tasks/inject-data-application/commands.yaml
+       ```       
+
+<!--
+2. List the running Pods:
+-->
 
-1. 获取一下当前正在运行的Pods信息：
+2. 获取正在运行的 pod
```
Comment (tengqm) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]: Don't use explicit numbers here

**Pull request merged**